cns/NetworkContainerContract.go

```diff
@@ -23,9 +23,9 @@ const (
 	DetachContainerFromNetwork               = "/network/detachcontainerfromnetwork"
 	RequestIPConfig                          = "/network/requestipconfig"
 	ReleaseIPConfig                          = "/network/releaseipconfig"
-	GetIPAddresses                           = "/debug/getipaddresses"
-	GetPodIPOrchestratorContext              = "/debug/getpodcontext"
-	GetHTTPRestData                          = "/debug/getrestdata"
+	DebugIPAddresses                         = "/debug/ipaddresses"
+	DebugPodContext                          = "/debug/podcontext"
+	DebugRestData                            = "/debug/restdata"
```

Comment:
This is a breaking change, I'm hoping scope is very limited?

Reply:
internal debug api to dump cns state, we're the only consumer